README.md

```diff
@@ -33,6 +33,7 @@ You can also run the tests in `fenics-adapter/tests` from this folder. Example:
 
 Add ``from fenicsadapter import Coupling`` in your FEniCS code. Please refer to the examples in the [tutorials repository](https://github.com/precice/tutorials) for usage examples.
 
+FEniCS configuration file`s name must be: ``precice-adapter-config.json`` and should be placed in the same directory as the solver in use.
```

Comment:
Better: set this name as default and provide option to use a custom name provided by the user.

Reply:
Current name and location is consistent with the `openfoam-adapter` approach (which I think doesn`t allow to change the name of `.yml` config files). Also, we would have to add one more parameter regarding the coupling to the solver.

Reply:
Agree.